src/styles/card.css

```diff
@@ -0,0 +1,83 @@
+:root {
+  --black-color: #060b35;
+  --gray-color: #616070;
+  --blue-color: #00acdc;
+  --white-color: #fff;
+}
```

Comment:
move it from the `card `block's file. better place it in the `normalize`, `index` or create separate file